apps/www/app/api-v2/submit-form-apply-to-supasquad/route.tsx

```diff
@@ -0,0 +1,188 @@
+import z from 'zod'
+
+const NOTION_API_KEY = process.env.NOTION_API_KEY
+const NOTION_DB_ID = process.env.NOTION_DB_ID
+
+const applicationSchema = z.object({
+  firstName: z.string().min(1, "First name is required"),
+  lastName: z.string().min(1, "Last name is required"),
+  email: z.string().email("Please enter a valid email address"),
+  tracks: z.array(z.string()).min(1, "Select at least 1 track"),
+  areas_of_interest: z.array(z.string()).min(1, "Select at least 1 area of interest"),
+  why_you_want_to_join: z.string().min(1, "This is required"),
+  monthly_commitment: z.number().min(1, "This is required"),
+  languages_spoken: z.array(z.string()).min(1, "Select at least 1 language"),
+  skills: z.string().min(1, "This is required"),
```

Comment:
- made skills optional
- should city, country be 2 fields?
- should monthly commitment be optional? what if I'm not sure, or "it depends"?

Reply:
> should city, country be 2 fields?
done, country is a dropdown and city a text field

> should monthly commitment be optional? what if I'm not sure, or "it depends"?
made it optional, and indicate on the description that "if not sure, leave blank"